Samsung/M31s/res/values/config.xml

```diff
@@ -586,32 +586,32 @@
         <item>420</item>
         <item>600</item>
     </integer-array>
-    <string-array name="config_tether_usb_regexs">
-        <item>rndis0</item>
-    </string-array>
     
     
     <bool name="config_fillMainBuiltInDisplayCutout">true</bool>
-    <bool name="config_hotswapCapable">true</bool>
     <bool name="config_showNavigationBar">true</bool>
-    <bool name="config_useDevInputEventForAudioJack">true</bool>
-    <bool name="config_wifi_dual_band_support">true</bool>
-    <bool name="config_wifi_fast_bss_transition_enabled">true</bool>
+    <bool name="config_supportAudioSourceUnprocessed">false</bool>
```

Comment:
I'm pretty confident that's wrong..

Reply:
Hmm🤔. Added those back now, and pushed that...